### PR TITLE
4x new features and 1x exception code are added.

### PR DESCRIPTION
**Pull request by JeesubKim**
[New features]
1. Scroll Zoom en/disable 
2. Control of Pitch and bearing, 
3. 3 of Map styles
4. Play map locations as a slides

[Exception code]
Undefined exception of props

### CODE REVIEW

File: src/MapBox.js [hunk 1]
```diff
@@ -42,85 +47,152 @@ function MapBox(props) {
...
+
       mMap.on('load', () =>{
         mMap.resize();
         setMap(mMap);
         
       });
+      
     };
 
     const drawMarkers = ()=>{
```
Comment (soaple) [inline src/MapBox.js]: Please add a space between () and =>, => and { for readability.

File: src/MapBox.js [hunk 2]
```diff
@@ -42,85 +47,152 @@ function MapBox(props) {
...
 
       if(map){
-        props.map.marker.forEach( (mark)=>{
-  
-          let reactMarker = React.createElement('div',getMarkerStyle(mark.img),'styled');
-          let marker = new mapboxgl.Marker(reactMarker).setLngLat(mark.coordinates).setPopup(new mapboxgl.Popup({offset:25,closeOnClick: mark.closeOnClick }).setHTML('<h3>'+mark.title + '</h3><p>'+mark.description +'</p>')).addTo(map);          
-          if(mark.display)
-            marker.getPopup().addTo(map);
-        })
+        if(props.map.marker!==undefined){
```
Comment (soaple) [inline src/MapBox.js]: Please add a space both sides of !== for readability.

**Review by soaple — CHANGES_REQUESTED** [review]
Please check the comments.

File: src/MapBox.js (hunk 1 above)
Comment (JeesubKim) [reply src/MapBox.js]: Okay, I added spaces in all of arrow function expression and comparison operators

jeesubKim/stickboard :  59f5945398f2b427615cb8d5442473071f3ef658

File: src/MapBox.js (hunk 2 above)
Comment (JeesubKim) [reply src/MapBox.js]: I added spaces in all of arrow function expression and comparison operators

jeesubKim/stickboard :  59f5945398f2b427615cb8d5442473071f3ef658

**Review by soaple — APPROVED** [review]
LGTM

**Pull request merged**